docs/lib/app.js

```diff
@@ -77,6 +78,15 @@ export default (locals, callback) => {
           </script>
           <script src="/assets/prism.js" data-manual></script>
           <script src="/bundle.js"></script>
+          <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
```

Comment:
Please update endpoint to use CDNJS

Please update end point to use CDNJS 

https://cdnjs.com/libraries/docsearch.js